### ISSUE TITLE
Migration failed

### ISSUE THREAD

**Issue by MyWay** [pocketbase/pocketbase#1579]
Hi, I'm trying to upgrade from 0.10.4 to 0.11.0, but I'm getting:

```
panic: Failed to apply migration 1673167670_multi_match_migrate.go: SQL logic error: no such column: options (1)
panic: Failed to apply migration 1673167670_multi_match_migrate.go: SQL logic error: no such column: options (1)
```

**Comment by ganigeorgiev** [comment]
Hm, that's strange.

Was your installation previously using < v0.8.0?

If yes, did you follow the migration guide from https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#migrate-from-v07x?

Or it is a clean v0.10.4 installation and then upgraded to v0.11.0 (I'm not able to reproduce it this way)?

**Comment by MyWay** [comment]
Yes, 0.7.2, can I proceed with these instructions for 0.8.0 using 0.10.4?

**Comment by ganigeorgiev** [comment]
> Or I could export collections and import directly on a fresh installation?

No, this will not work since there are some changes to the db structure.

---

> 0.7.2, can I proceed with these instructions for 0.8.0 using 0.10.4?

Yes, it should work. The temp `upgrade` command is available in the v0.10.4 binary.

**Make sure to have a backup somewhere safe of your `pb_data` before doing any changes.**
